contracts/NonFungibleToken/NonFungibleMintableBurnable.aes

```diff
@@ -0,0 +1,86 @@
+@compiler >= 6
+
+include "Option.aes"
+
+contract NonFungibleMintableBurnable =
+
+  record state =
+    { name: string
+    , symbol: string
+    , owner: address
+    , map_owned_tokens_count: map(address, int)
+    , map_token_owner: map(int, address)
+    , map_token_approvals: map(int, address)
+    , map_operator_approvals : map(address, map(address, bool)) }
```

Comment:
I think this can be simplified. Actually lets first merge the iris compatibility and then we'll work on the contracts.

Reply:
not sure if it's worth it as there is currently an active grant running to tackle the NFT standard. but I agree this could be simplified. I didn't want to put more time into that stuff right now. at least it works and the tests succeed ;-)

Reply:
Sure! 

My point was for all of the contracts we can additionally revisit each and every one of them and optimize/clean them. So I won't focus on this now, rather will check the aeproject config and merge the PR so that we can use the CI for later improvements.